test/node_modules/side-effects-array/package.json

```diff
@@ -1,6 +1,7 @@
 {
 	"main": "./index.js",
 	"sideEffects": [
+		"./index.js",
```

Comment:
Why are you including the index? I think it is very instructive to see that in case a side-effect-free file reexports from side-effect-ful file, only the latter file is included.
Otherwise thanks so much for spotting this. It goes to show that testing does not mean everything is working as intended if the test is bananas 🙄

Reply:
If I remove `index.js` from the `sideEffects` array, then none of the `array-` files are included:

```
      AssertionError [ERR_ASSERTION]: Input A expected to strictly deep-equal input B:
+ expected - actual

  [
    'false-dep1',
    'true-dep1',
    'true-dep2',
-   'true-index'
+   'true-index',
+   'array-dep1',
+   'array-dep3',
+   'array-dep5',
+   'array-index'
  ]
```

Is this expected or might this be another bug?